PULL-REQUESTS.md

```diff
@@ -0,0 +1,86 @@
+# Pull Request Checklist
+
+## Beginning the Work
+Good Pull Requests start well before you write your first line of code. Be prepared to craft discreet, focused and intentioned PR's by following a few simple guidelines:
+
+1. Create a __new__ branch
+1. Select a branch name that matches your company's guidelines
+1. In the absence of guidelines, use a consistent, meaningful branch naming convention. Consider using the story or task identifier in the branch name.
+1. Keep the work to a single feature / story / task to limit the scope of the upcoming PR
+1. Use meaningful commit messages:
+  * Great: `ABC-123: Updated logic to include the correct coefficient multiplier based on the information provided by @teammate in our grooming session on 10/02`
```

Comment:
I would consider following the 80 character limit when possible for the first line of your commit. You can put longer messages on the next couple. Maybe something like this:
```
ABC-123: Updated logic to include the correct coefficient multiplier

- based on the information provided by @teammate in our grooming session on 10/02
```

Reply:
Yeah, that's a good catch. I didn't realize I'd hit 80 chars in that line.

Reply:
I figured if we're going to be giving them advice, we should not uh give them too terrible of advice in some other manner. :)